plugins/meta/firewall/firewall.go

```diff
@@ -46,8 +46,27 @@ type FirewallNetConf struct {
 	// the firewalld backend is used but the zone is not given, it defaults
 	// to 'trusted'
 	FirewalldZone string `json:"firewalldZone,omitempty"`
+
+	// IngressPolicy is an optional ingress policy.
+	// Defaults to "open".
+	IngressPolicy IngressPolicy `json:"ingressPolicy,omitempty"`
 }
 
+// IngressPolicy is an ingress policy string.
+type IngressPolicy = string
+
+const (
+	// IngressPolicyOpen ("open"): all inbound connections to the container are accepted.
+	// IngressPolicyOpen is the default ingress policy.
+	IngressPolicyOpen IngressPolicy = "open"
+
+	// IngressPolicySameNetwork ("same-network"): connections from the same network are accepted, others are blocked.
+	// This is similar to how Docker libnetwork works.
+	// IngressPolicySameNetwork executes `iptables` regardless to the value of `Backend`.
+	// IngressPolicySameNetwork may not work as expected for non-bridge networks.
+	IngressPolicySameNetwork IngressPolicy = "same-network"
```

Comment:
Someday in the future, it might be nice to have L3 isolation, rather than relying on the bridge interface (e.g. for p2p "networks").

What if we call this "same-bridge" for now? That will make it explicit that this is dependent on bridges.

Reply:
Thanks fore reviewing, renamed `same-network` to `same-bridge`. PTAL.